subprojects/plugins/src/integTest/groovy/org/gradle/java/compile/BasicJavaCompilerIntegrationSpec.groovy

```diff
@@ -218,11 +221,11 @@ compileJava {
 """
     }
 
-    @Requires(UnitTestPreconditions.Jdk9OrLater)
     def "compile with release flag using #notation notation"() {
         given:
         goodCode()
         buildFile << """
+java.toolchain.languageVersion = JavaLanguageVersion.of(11)
```

Comment:
❌ I don't think we can do this because classes that extend this test are expecting that the compiler runs in different ways (in-process or CLI or daemon).

Instead of using a toolchain here, let's leave this to run on JDK9+ and move up to release=9

🤔 Change this to run on JDK11+ and move release=11  

I think the target compatibility is actually ignored.

Reply:
I'll do 11, since that's the minimum we run on CI above 8 anyways.